example_dot_env

```diff
@@ -1,3 +0,0 @@
-SESSION_SECRET='1845f5419299977a57a61e74c659ad804dca6f7fd5209b3888e773f123b113f8'
-
-SUNLIGHT_FOUNDATION_KEY='55a27bdc46b947c4b63b791b7cf6fa2f'
```

Comment:
@nhuntwalker if you remove this file, the line `cp example_dot_env .env` in the above docs will no longer work. I'd say leave this for now, and let's make a separate issue for removing the keys from the repo.